api/soap/mantisconnect.wsdl

```diff
@@ -163,6 +164,8 @@
    <xsd:element name="monitors" type="tns:AccountDataArray" minOccurs="0"/>
    <xsd:element name="sticky" type="xsd:boolean" minOccurs="0"/>
    <xsd:element name="tags" type="tns:ObjectRefArray" minOccurs="0"/>
+   <xsd:element name="alias" type="xsd:string" minOccurs="0"/>
```

Comment:
Yes, it is not strictly necessary, but wouldn't it be an additional flexibility?